lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -678,24 +676,24 @@ def create_default_securitygroup(self, resource_group, location, name, os_type,
             self.check_provisioning_state(group)
             return group
 
-        parameters = NetworkSecurityGroup()
+        parameters = self.network_models.NetworkSecurityGroup()
```

Comment:
Why we fix the version in the code? That means we should use all the azure SDK model in this way not only in the azure_rm_common